framework/src/utils/SlepcSupport.C

```diff
@@ -36,7 +40,7 @@ InputParameters
 getSlepcValidParams(InputParameters & params)
 {
   MooseEnum solve_type("POWER ARNOLDI KRYLOVSCHUR JACOBI_DAVIDSON "
-                       "NONLINEAR_POWER NEWTON PJFNK JFNK",
+                       "NONLINEAR_POWER NEWTON PJFNK PJFNKMO JFNK",
```

Comment:
I do not like this name because this is not Jacobian free in sense that we assemble both A and B matrix. Do we have a test for `NEWTON`? If we do not we should add one. I personally think `NEWTON` is not that useful and suggest to name this type as `NEWTON`. Or we can name this type as `MONK` (monk? ;-)) or `PMONK`.

Reply:
I like the name I have right now

Not sure if we should keep arguing on this